### DISCUSSION TITLE
Part cooling fan PWM

### DISCUSSION THREAD

**Discussion by asychev** [Jyers/Marlin#876] [General Discussion]
Hi everyone!

I'm struggle to find correct information, so my hope I could fine experienced guys here :)

My question is about part cooling fan on Creality 32bit boards. It has 2 wires and it is able to change speed - how this really works? I search a lot and my current understanding is this:

- It is not a PWM (real PWM usually 3 or 4 wires)
- Fan speed managed by connecting/disconnecting GROUD wire - is this correct?

So, for example, If I want to use Noctua 12V FLX (3 wire) as a part fan:
- I need a step down 24V->12V converter: input plus must be connected to 24V plus (from board or PSU); input ground must be connected to common ground (from board or PSU)
- noctua fan PLUS must be connected to converter plus output
- noctua fan GND must be connected to Creality board part cooling fan GND
- 3rd noctua wire must be disconnected

This way part cooling fan will be permanently powered by 12V from converter and GND will be "managed" by Creality fake PWM. Is this correct understanding?

Any additional information is much appreciated.

**Comment by smooreace** [comment]
It's much easier to run 2 12v fans. It will give far better cooling and
eliminate all the BS gimmicks for power management.,

**Reply by asychev** [reply]
I agree, but what shroud do you recommend in this case?

**Reply by agalakhov** [reply]
Fans have a resistance that changes periodically with rotation and thus should not be connected in series. They may work incorrectly or even be damaged.

**Comment by smooreace** [comment]
I use the Petsfang personally, and my fans never need to be on higher than
30% for PLA

**Reply by asychev** [reply]
So, you have nothing to say about the topic, right? Thanks.

**Reply by gothcha** [reply]
@asychev The part fan is controlled by a MOSFET in the ground lead of the fan. This is pulsed ON/OFF using the PWM output from the STM micro to create an average reduced voltage for the 24V fan. 
 
@smooreace is suggesting that if you use two 12V fans wired in series then this does work with the PWM and does not require any additional circuitry.

**Reply by asychev** [reply]
@gothcha thanks. Two fans in series is heavily limiting choices of ducts/fangs, for example Satsana or Briss does not support it. What else I could look for?

**Reply by gothcha** [reply]
Sorry I am not aware of a simple solution to use 12V fans. There are some 24v fans that may be quieter I guess

**Reply by rorywquin** [reply]
`YES` what you said below.....

I need a step down 24V->12V converter: input plus must be connected to 24V plus (from board or PSU); input ground must be connected to common ground (from board or PSU)
noctua fan PLUS must be connected to converter plus output
noctua fan GND must be connected to Creality board part cooling fan GND
3rd noctua wire must be disconnected

HOWEVER, I replaced my MB, PSU & hot end fans and kept the blower fan. I used the Satsana shroud. It has teh effect that my printer is silent. The worst noise is the MB & PSU

**Comment by Jyers** [comment]
Just a note, it is really PWM. I believe what you are considering "real PWM" is a fan that uses PWM as a control signal rather than PWM for power. PWM is just the means with which the voltage is varied (pulse width modulation). On 3/4 pin fans, this signal goes to the controller on the fan and tells it how much gas to give it, where as on the parts cooling fan of the printer, the full 24v is being modulated, allowing you to vary the voltage to the fan with only two wires.

**Reply by t3chguy** [reply]
the 3rd pin on 3 pin fans typically is a tacho line rather than a PWM line. PWM on a 3 pin w/ tacho fan can actually mess with the reading. 3 pin fans are typically voltage-controlled rather than pulse-controlled.

**Reply by Jyers** [reply]
Hmmm, interesting. Haven't worked a lot with 3 pin fans. Usually I just see 4 pin ones with V+, PWM, Tach, Ground

**Reply by asychev** [reply]
@Jyers Yes, you are right, thanks for note

**Comment by scaramonga** [comment]
Watch this video.  It explains it all rather nicely ;)
https://www.youtube.com/watch?v=k9Yy8OxohGI

**Comment by agalakhov** [comment]
Do NOT connect 12V fans in series. Motors aren't lightbulbs. They won't work well.

Use a DC-DC step-down. Your wiring is correct (input of the DC-DC to 24V, output + to fan Pin 2, Fan "GND" of Creality board to fan Pin 1, fan Pin 3 unconnected).

**Comment by smooreace** [comment]
Funny... they work perfectly. So much so that they are within 5-10 RPM of
each other at all tested ranges

**Reply by agalakhov** [reply]
They will rotate and even at the same speed but there will be voltage spikes on each of them. Instead of 12V+12V the voltage will quickly jump between 0V+24V and 24V+0V with 12V+12V as average. This isn't healthy for the fan.

**Reply by scaramonga** [reply]
Yeah, they do ramp up and down when in series, OK for non-critical cooling, but not for 3D printing :) Yeah you will get away with it, but it's bad on the components (fans) and also the wiring/circuitry that's feeding them ;)

**Reply by agalakhov** [reply]
The worst thing here is, the 12V fan might get full 24V sometimes. Not all fans can withstand this.

**Comment by smooreace** [comment]
They most certainly don't cycle in RPM at all. And IF they are power
cycling then they are faster than $1000 fluke can see. And certainly isn't
relevant to the wires!

**Comment by smooreace** [comment]
To take that even further.... there is no logic in the slightest that can
support the motors jumping back and forth between one being powered and
then the other either. MAYBE if you somehow managed to get 2 BRUSHED motors
to spin exactly opposite timed from each other. But 1... that insanely rare
instance would only happen for a fraction of a second once in an insanely
rare while, and 2... if you're using brushed motors then you might as well
go back to lincoln logs. Because this is WAY over your head!

**Reply by rorywquin** [reply]
@smooreace 

Perhaps your comment ......

"It's much easier to run 2 12v fans. It will give far better cooling and
eliminate all the BS gimmicks for power management.,”

....raised a few hackles!

1)  Using a buck converter is not a BS gimmick . They are widely used for DC>DC voltage reduction applications. 
2)  Most people do not want or need a two fan part cooling setup.

I (like thousands of others) have had no problems printing with a single part cooling fan.  Using 2 fans to get a round the need for a buck converter seems illogical.

**Reply by agalakhov** [reply]
One DC-DC handles two fans perfectly.

**Reply by rorywquin** [reply]
I used two as it was easier/neater to install them where I wanted them rather, than run wires. They cost peanuts.

**Comment by smooreace** [comment]
Running a BC the way the vast majority of people are running them is most
definitely NOT providing the assumed voltage or the expected fan output.
Both of the Creality boards use PWN output to modify fan output percentage.
Pushing that PWM signal through a BC leads to a much higher fan RPM over
any given time than the chosen output because the BC will do everything it
can to maintain the chosen voltage... effectively nullifying that "choice".

As far as a 12v fan being hit with 24v goes.... bullshit! You show me
actual video and scope output of that happening with brushless fans being
driven by a Creality board and Ill eat my words. Until then... crack is bad
for ya!

**Reply by rorywquin** [reply]
PSU fan and MB fans are on/off so no issues connecting a 12v fan via a BC. Part cooling fan is speed controlled and mine is still the stock 24v blower. However easy enough to wire a PWM controlled fan with a BC (via ground line) and still have control on a 12v fan.

Don’t need a $1000 Fluke meter to tell me this works.

**Reply by agalakhov** [reply]
The BC is not to be connected to the fan output. It has to be powered from 24V directly. The BC provides +12V for the fan, and GND for the fan is taken from the fan output of the Creality board where it is connected to the drain of PWM MOSFET.

**Reply by rorywquin** [reply]
@agalakhov 

My PSU fan (80mm) is directly plugged to the 12v Fan supply (on/off). My MB (also 80mm) fan is MainboardFanSocket>BC>Fan (on/off).  Hot end cooling is GreenHotEndConnector>BC>12vFan (always on).

All been running like that for 10 months printing daily no trouble.

I have not upgraded the blower yet  & I find cooling performance fine with the Satsana duct (but if I did, the method you have described is correct to get the speed control). 

Last week I printed one of these (V2 difussers) - no supports & no problems.

https://www.thingiverse.com/thing:4812520

**Comment by smooreace** [comment]
I would hope you wouldnt need support! That should print fine with no
cooling at all

**Reply by rorywquin** [reply]
Please do one and let us know! :-)
